api.md

```diff
@@ -1835,16 +1797,16 @@ all the updates to the hypertable.
 
 ---
 
-## DROP VIEW (Continuous Aggregate) :community_function: [](continuous_aggregate-drop_view)
-Continuous aggregate views can be dropped using `DROP VIEW` statement.
+## DROP MATERIALIZED VIEW (Continuous Aggregate) :community_function: [](continuous_aggregate-drop_view)
+Continuous aggregate views can be dropped using `DROP MATERIALIZED VIEW` statement.
```

Comment:
This was partially updated as a result of changes to the Using chapter. I decided to move all the changes to a separate PR since I expected large changes.

Reply:
Updated the section on `DROP MATERIALIZED VIEW` in #496. I have tested that it works without `CASCADE`.  It is not pushed yet.